tests/fixtures/optimizer/annotate_functions.sql

```diff
@@ -323,18 +323,6 @@ DOUBLE;
 CEIL(tbl.bignum_col);
 BIGDECIMAL;
 
-# dialect: bigquery
-FLOOR(1);
-DOUBLE;
-
-# dialect: bigquery
-FLOOR(5.5);
-DOUBLE;
-
-# dialect: bigquery
-FLOOR(tbl.bignum_col);
-BIGDECIMAL;
```

Comment:
Was this an oversight? Removing them doesn't look right.

Reply:
These tests are not the same as those added in #6030 though, they're targeting BigQuery.